### PR TITLE
Document release notes process changes

### PR DESCRIPTION
**Pull request by keirthana**
* Rename announcements folder to release-notes
* Add release notes template to the repo

### CODE REVIEW

File: release-notes/release-notes-template.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Other
+
+< In addition to the below placeholder points, list minor features that are part of the release but were not elaborated as separate sections >
+
+* Android security updates for < insert-month-year > (see Android Security Bulletin - < insert-month-year along with the link to bulletin > for more information).
+* The Android WebView has been updated to < insert-version-along-with-link >.
+
+
+### Removed functionality
```
Comment (ajanon) [inline release-notes/release-notes-template.md:34]: Should this be a subsection of `New features & improvements` or should it be its own section?

File: release-notes/release-notes-template.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+< List features that were part of Anbox Cloud so far or that were on the deprecation path for a while and have been removed for the current release >
+
+
+### Deprecations
```
Comment (ajanon) [inline release-notes/release-notes-template.md:39]: Same as above: should this be a subsection of `New features & improvements` or should it be its own section?

File: release-notes/release-notes-template.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Feature 2
+
+< Explain feature 2 >
+
+
+< Add feature sections as necessary >
+
+
+### Dashboard improvements
```
Comment (adglkh) [inline release-notes/release-notes-template.md:21]: I compared this with the 1.18.0 md, there are
- Streaming improvements
- Dashboard improvements

Are we flexible about the sub-sections under the `New features & improvements` or we want to have fine-grinded sub-sections for each portion in our stack?

File: release-notes/release-notes-template.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+< List enhancements to the dashboard here. You can screenshots to illustrate something new or enhanced. >
+
+
+### Other
+
+< In addition to the below placeholder points, list minor features that are part of the release but were not elaborated as separate sections >
+
+* Android security updates for < insert-month-year > (see Android Security Bulletin - < insert-month-year along with the link to bulletin > for more information).
+* The Android WebView has been updated to < insert-version-along-with-link >.
```
Comment (adglkh) [inline release-notes/release-notes-template.md:31]: This reminds me of one thing: looks like we forgot to mention the update to Android Webview in our 1.18.0 release note.

File: release-notes/release-notes-template.md (hunk 1 above)
Comment (jat-canonical) [reply release-notes/release-notes-template.md:34]: I think this is fine as a separate topic generally deprecated/removed features/functionalities carry a higher risk for people upgrading from lower versions and hence needs a little more importance. What does everyone else think ?

Comment (keirthana) [reply release-notes/release-notes-template.md:34]: I think they should be separate sections - reason is as Jatin has explained. Removed functionality and deprecations need to be prominent enough for users to take notice.

File: release-notes/release-notes-template.md (hunk 2 above)
Comment (keirthana) [reply release-notes/release-notes-template.md:39]: See above answer

File: release-notes/release-notes-template.md (hunk 3 above)
Comment (keirthana) [reply release-notes/release-notes-template.md:17]: I think it is worth having separate stack headings in the template. When we actually write the release notes, if we don't have significant improvements to one stack, we can always remove the section. So for now, I have categorised as Core stack improvements, Streaming improvements, Dashboard improvements.

File: release-notes/release-notes-template.md (hunk 4 above)
Comment (keirthana) [reply release-notes/release-notes-template.md:26]: I will update the release notes for 1.18.

File: release-notes/release-notes-template.md (hunk 2 above)
Comment (ajanon) [reply release-notes/release-notes-template.md:32]:
```suggestion
## Deprecations
```

File: release-notes/release-notes-template.md (hunk 1 above)
Comment (ajanon) [reply release-notes/release-notes-template.md:28]:
```suggestion
## Removed functionality
```

**Review by ajanon — CHANGES_REQUESTED** [review]

**Review by ajanon — APPROVED** [review]
LGTM

**Pull request merged**